ui.R

```diff
@@ -1,794 +1,797 @@
-         HTML('<iframe width="560" height="315" src="https://www.youtube.com/embed/b-fYoUdksRo" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>'),
-         HTML('<iframe width="560" height="315" src="https://www.youtube.com/embed/g0n5yxQ4Z34" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>')
-         )
-,
-
-
-###################################
-### Tab 6 - trouble shooting ###
-###################################
-
-tabPanel(id="trouble", "Troubleshooting",
-#          tags$div(
-# includeHTML("troublesh.html")
-#          )
-tags$iframe(style="height:1500px; width:100%; scrolling=yes",
-            src="ts2.pdf")
-),
-
-
-
-
-
-###################################
-### Tab 7 - Full update history ###
-###################################
-
-tabPanel(id="update", "Full update history",
-         tags$iframe(style="height:1500px; width:100%; scrolling=yes",
-                     src="fullupdatehistory4.pdf")
-),
-
-
-
-##############################
-### Tab 8 - Privacy notice ###
-##############################
-
-
-
-tabPanel(id="privacy", "Privacy notice",
-   tags$iframe(style="height:1500px; width:100%; scrolling=yes",
-               src="gdpr.pdf")
-)
-
-)
-)
-)
+###### Combined MEtaInsight ######
+
+#install.packages(c("dplyr","metafor", "netmeta","shiny", "shinyAce","rmarkdown", "knitr", "shinydashboard", "gemtc"
+#  , "shinyalert", "ggplot2", "plotly"))
+
+# install.packages("pkgbuild")
+# pkgbuild::has_build_tools()
+# install.packages(c("remotes", "knitr", "devtools"))
+# remotes::install_github("audrey-b/BUGSnet@v1.0.4", upgrade = TRUE, build_vignettes = TRUE)
+# devtools::install_github("audrey-b/BUGSnet@v1.0.4", upgrade = TRUE, build_vignettes = TRUE)
+
+library(metafor)
+library(netmeta)
+library(shiny)
+library(shinyAce)
+library(rmarkdown)
+library(knitr)
+library(shinydashboard)
+library(gemtc)
+library(shinyalert)
+library(ggplot2)
+library(plotly)
+library(shinyWidgets)
+library(shinyjs)
+library(shinyBS)
+library(cowplot)
+library(shinycssloaders)
+
+
+source("PlotFunctionsRKO.R", local = TRUE) # Plot functions
+load("blank.rds") # Objects to store data for plot functions
+
+dashboardPage(
+  dashboardHeader(disable = TRUE),
+  dashboardSidebar(disable = TRUE),
+  dashboardBody(
+  tags$head(
+    # load custom stylesheet
+    # To ensure white background and no horizontal scroll bars on ranking panel
+    tags$link(rel = "stylesheet", type = "text/css", href = "app.css"),
+    includeHTML("www/favicon/favicon.html"),
+    tags$meta(name="description", content="A interactive web tool for network meta-analysis (NMA) that leverages established analysis routines"),
+    tags$meta(name="keywords", content="MetaInsight, NMA, Network Meta Analysis, App"),
```

Comment:
do we want/need "meta analysis" as a separate keyword, or does it already match due to it being a subset?

Reply:
They should probably be split, but most search engines ignore them anyway, but they are useful to know what keywords you should be targeting.

Reply:
I have split these now.